package.json

```diff
@@ -1,27 +1,27 @@
 {
+  "$schema": "./node_modules/ng-packagr/package.schema.json",
   "name": "ng-gapi",
   "version": "0.0.93",
   "description": "Angular 9+ Google api module ng-gapi",
-  "main": "./lib/index.js",
-  "files": [
-    "lib/*"
-  ],
-  "typings": "./lib/index.d.ts",
+  "ngPackage": {
+    "lib": {
+      "entryFile": "src/index.ts"
+    },
+    "whitelistedNonPeerDependencies": [
+      "@types/gapi",
+      "@types/gapi.auth2",
+      "tslib"
+    ]
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "ng-packagr -p package.json",
     "tsc": "ngc"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/rubenCodeforges/ng-gapi"
   },
   "keywords": [
-    "Angular2",
```

Comment:
Can you please revert the keywords back , they are important for indexing

Reply:
Done